package.json

```diff
@@ -27,8 +27,9 @@
     "prepare-preview-builds": "./scripts/prepare-preview-builds.sh",
     "publish-previews": "yarn workspaces foreach --parallel --verbose run publish:preview",
     "setup": "yarn install",
-    "test": "yarn workspaces foreach --parallel --verbose run test",
-    "test:clean": "yarn workspaces foreach --parallel --verbose run test:clean && yarn test"
+    "test": "yarn test:verbose --collectCoverage=false --reporters=jest-silent-reporter",
+    "test:clean": "yarn workspaces foreach --parallel --verbose run test:clean && yarn test",
```

Comment:
This is a good change as it makes this script consistent with how `build:clean` works.

However, what do you think about moving the `&& yarn test` to the package level? Then you could keep this simple:

``` suggestion
    "test:clean": "yarn workspaces foreach --parallel --verbose run test:clean",

Reply:
This doesn't work because the package-level `test:clean` scripts running in parallel means concurrent read/writes to the jest cache directory, which causes errors. So the `jest --clearCache` for every package needs to run first before the actual tests can be executed.

Reply:
Maybe we could just do `"test:clean": "jest --clearCache && yarn test"`, although we'll need to install `jest` in root for this.

Reply:
No, this is fine. And I see this script was already here anyway, you just added another after this. So all good here.